src/OpenTelemetry.Exporter.GoogleCloud/GoogleCloudTraceExporter.cs

```diff
@@ -7,14 +7,14 @@
 using Google.Api.Gax.Grpc;
 using Google.Cloud.Trace.V2;
 using Grpc.Core;
-using OpenTelemetry.Exporter.Stackdriver.Implementation;
+using OpenTelemetry.Exporter.GoogleCloud.Implementation;
 
-namespace OpenTelemetry.Exporter.Stackdriver;
+namespace OpenTelemetry.Exporter.GoogleCloud;
 
 /// <summary>
 /// Exports a group of spans to Stackdriver.
 /// </summary>
-public class StackdriverTraceExporter : BaseExporter<Activity>
+public class GoogleCloudTraceExporter : BaseExporter<Activity>
```

Comment:
You may want to also deprecate this going forward.

Google Cloud Trace now supports OTLP natively: https://cloud.google.com/stackdriver/docs/reference/telemetry/overview

Reply:
@jsuereth, @bielu, if the package is no longer needed, I would change the PR, add only `Obsolete` attributes, and update CHANGELOG with notification that it is last PR, and there will be no more updates to the package.

Next step is a release, and full removal of the code from the repository (except some README/CHANGE:LOG).

What is your perspective on this? Renaming package, and obsoleting it shortly does not seems reasonable to me.

Reply:
I think marking this obselete and letting people know they can use the built-in OTLP exporter is definitely the way forward.

Regarding when to drop the package, that's a decision for .NET maintainers on how long to provide support and migration to users of OpenTelemetry packages, but I think one release then gone seems a bit aggressive.

@bielu I'm happy to help with docs, although @aabmass, @dashpole  or @psx95 may be better suited to help you there.

Reply:
We can try to get some GCP docs written, although it may take us a bit to prioritize.  I think we would need to add a sample to https://github.com/GoogleCloudPlatform/dotnet-docs-samples, similar to https://github.com/GoogleCloudPlatform/golang-samples/tree/main/opentelemetry/instrumentation, and then we can write the docs page internally.

Reply:
@Kielek will raise pr, however there is no docs yet, so I guess we can raise pr to update and obsolete and later second one to link to docs when they are up?